### PR TITLE
ARROW-12477: [Release] Download aarch64 miniforge

### PR DESCRIPTION
**Pull request by cyb70289**

### CODE REVIEW

**Comment by cyb70289** [comment]
Travis CI is green https://travis-ci.org/github/cyb70289/arrow/builds/767705246

File: dev/release/verify-release-candidate.sh [hunk 1]
```diff
@@ -224,6 +224,8 @@ setup_miniconda() {
     else
         MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
     fi
+  elif [ "$(uname)" == "Linux" ] && [ "$(uname -m)" == "aarch64" ]; then
+    MINICONDA_URL=https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
   else
     MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
```
Comment (kszucs) [inline dev/release/verify-release-candidate.sh:230]: Shall we use miniforge form amd64 as well?

**Comment by kszucs** [comment]
Thanks @cyb70289 for this PR! Have you managed to successfully verify the release on arm64?

**Comment by cyb70289** [comment]
> Thanks @cyb70289 for this PR! Have you managed to successfully verify the release on arm64?

Yes. I verified C++ and Python on Ubuntu-18.04 AArch64. It works correctly.

File: dev/release/verify-release-candidate.sh (hunk 1 above)
Comment (cyb70289) [reply dev/release/verify-release-candidate.sh:230]: Using miniforge for all platforms looks better to me.
Maybe after 4.0 release? I don't know.

**Review by kszucs — APPROVED** [review]

**Review by kou — APPROVED** [review]
+1

**Pull request closed without merging**